frontend/public/locales/en/translation.json

```diff
@@ -1877,6 +1883,7 @@
   "name": "name",
   "Name": "Name",
   "Name needs to be unique to the namespace on each of the managed clusters.": "Name needs to be unique to the namespace on each of the managed clusters.",
+  "Name specify the network attached to the nodes in the format \"[namespace]/[name]\" to reference the multus network attachment definition": "Name specify the network attached to the nodes in the format \"[namespace]/[name]\" to reference the multus network attachment definition",
```

Comment:
Typo here? I think it's supposed to be "The name specifies the network..."

Reply:
This is marked as resolved but not fixed?